doc/examples.xml

```diff
@@ -711,12 +711,15 @@ true
   <Oper Name="BookGraph" Arg="[filt, ]m"/>
   <Returns>A digraph.</Returns>
   <Description>
-    The <E>Book digraph</E> is the Cartesian product of a complete digraph with 2 vertices
-    (as the "Book Spine") and the <M><A>m</A>+1</M> star digraph (as the "Pages").
-    For more details on the Book digraph please refer to <URL>https://mathworld.wolfram.com/BookGraph.html</URL>.<P/>
-
-    See <Ref Func="DigraphCartesianProduct" Label="for a positive number of digraphs"/>,
-    <Ref Oper="CompleteDigraph"/>, and <Ref Oper="StarGraph"/>. <P/>
+    The <E>Book digraph</E> is the Cartesian product of a complete digraph with
```

Comment:
Reminder to self: graph/digraph naming mismatch

Reply:
I'll make a separate note to myself to check the manual for things like this and fix them. (Both the operation being 'BookGraph' but the documentation saying 'Book digraph', and also the use of capitalisation in this entry.